jupyterlab_bigquery/src/utils/BigDataManager.ts

```diff
@@ -0,0 +1,34 @@
+class BigDataManager {
```

Comment:
I don't know if I like the name BigDataManager. How about `QueryResultsManager` or just `DataManager` if it is generic.

Reply:
I was intended to be a generic class, but I found different info need different format.  I renamed it to `QueryResultsManager`.